[BUG] 无法打开生物识别认证功能

### 问题描述


在设置中打开生物识别认证时应用崩溃
```
FATAL EXCEPTION: main
Process: com.rosan.installer.x.revived, PID: 27233
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.rosan.installer.x.revived/com.rosan.installer.ui.activity.EmptyFragmentActivity}: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3215)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3319)
	at android.app.ActivityThread.-wrap12(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1788)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:167)
	at android.app.ActivityThread.main(ActivityThread.java:7145)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:820)
Caused by: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
	at l.t.l(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:671)
	at l.g.setContentView(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:3)
	at l.g.onCreate(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:12)
	at android.app.Dialog.dispatchOnCreate(Dialog.java:411)
	at android.app.Dialog.show(Dialog.java:304)
	at k5.p.C(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:11)
	at k5.n0.n(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:46)
	at k5.n0.j(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:71)
	at k5.h0.P(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:62)
	at k5.h0.u(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:36)
	at com.rosan.installer.ui.activity.EmptyFragmentActivity.onStart(r8-map-id-2842b67c1879cd3350bcf92e1ff73187c47c5ce21675e3cfbc6723ac3ea29eaa:60)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1335)
	at android.app.Activity.performStart(Activity.java:7322)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3168)
	... 9 more
```

### 问题复现


1. 前往安装器设置/卸载器设置
2. 打开“安装时需要通过设备生物认证”/“卸载时需要通过设备生物认证”
3. 应用崩溃

### 预期行为


能够成功打开生物识别认证功能

### 手机信息


系统：Android 8.1
InstallerX 版本：latest alpha (2.3.2.41ca968)

### 屏幕截图或录屏


_No response_

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -141,7 +141,7 @@
             android:name=".ui.activity.BiometricsAuthenticationActivity"
             android:exported="false"
             android:launchMode="singleInstance"
-            android:theme="@style/Theme.Installer.Translucent"
+            android:theme="@style/Theme.Installer.Translucent.AppCompat"
             android:excludeFromRecents="true"
             android:finishOnTaskLaunch="true" />
 

--- a/app/src/main/res/values-night/themes.xml
+++ b/app/src/main/res/values-night/themes.xml
@@ -2,4 +2,12 @@
 <resources>
 
     <style name="Theme.Installer" parent="android:Theme.Material.NoActionBar" />
+
+    <style name="Theme.Installer.Translucent.AppCompat" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:windowIsTranslucent">true</item>
+        <item name="android:windowBackground">@android:color/transparent</item>
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+    </style>
+
 </resources>

--- a/app/src/main/res/values/themes.xml
+++ b/app/src/main/res/values/themes.xml
@@ -8,4 +8,12 @@
         <!--沉浸状态栏和导航栏-->
         <item name="android:windowBackground">@android:color/transparent</item>
     </style>
+
+    <style name="Theme.Installer.Translucent.AppCompat" parent="Theme.AppCompat.Light.NoActionBar">
+        <item name="android:windowIsTranslucent">true</item>
+        <item name="android:windowBackground">@android:color/transparent</item>
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+    </style>
+
 </resources>

